docs/csharp/language-reference/builtin-types/integral-numeric-types.md

```diff
@@ -61,7 +61,9 @@ The `nint` and `nuint` types in the last two rows of the table are native-sized
 
 The native-sized integer types are represented internally as the .NET types <xref:System.IntPtr?displayProperty=nameWithType> and <xref:System.UIntPtr?displayProperty=nameWithType>. Starting in C# 11, the `nint` and `nuint` types are aliases for the underlying types.
 
-The default value of each integral type is zero, `0`. Each of the integral types except the native-sized types has `MinValue` and `MaxValue` constants that provide the minimum and maximum value of that type.
+The default value of each integral type is zero, `0`.
+
+Each of the integral types has `MinValue` and `MaxValue` properties, that provide the minimum and maximum value of that type. These properties are compile-time constants, except for the case of the native-sized types (`nint and `nuint), for which `MinValue` and `MaxValue` are calculated at runtime, and depend on whether the running process is 32-bit or 64-bit.
```

Comment:
This is missing one closing code fence tick. Also, unnecessary commas. I split the sentences into multiple sentences to make it easier to understand. Finally, I removed the references to 32 bit and 64 bit. (The sizes are also different on IoT devices that could be different sizes. 

```suggestion
Each of the integral types has `MinValue` and `MaxValue` properties that provide the minimum and maximum value of that type. These properties are compile-time constants except for the case of the native-sized types (`nint and `nuint`). The `MinValue` and `MaxValue` properties are calculated at runtime for native-sized types. The sizes of those types depend on the processor.
```